### PR TITLE
Feat: Swiggy clone is added

### PR DESCRIPTION
**Pull request by Madhavan112**
Closes: #564 

- **Project Name** : Swiggy Clone
- **Name:** : Madhavan V
- **Idenitfy yourself:** : GSSOC24

### Describe the add-ons or changes you've made 📃
Added a Swiggy Clone Project in React Section

### How Has This Been Tested? ⚙️
My project has been checked locally and browser compatibility is checked

### Screenshots 📷

Swiggy-Clone

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: React-JS-Projects/Basic/Swiggy-Clone/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+<img src='./screenshot.webp'>
+
+
+## :zap: Working Video 📹
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/Swiggy-Clone/README.md:73]: Remove this heading, If video is not attached.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Resolve merge conflicts.

File: React-JS-Projects/Basic/Swiggy-Clone/README.md (hunk 1 above)
Comment (Avdhesh-Varshney) [reply React-JS-Projects/Basic/Swiggy-Clone/README.md:73]: Properly attach your issue with pr.

**Comment by Madhavan112** [comment]
Compeleted requested changes

**Comment by Madhavan112** [comment]
> Resolve merge conflicts.

I don't have write access so i cannot resolve this conflict

**Pull request merged**